cuebot/src/test/resources/opencue.properties

```diff
@@ -15,3 +15,10 @@ grpc.rqd_server_port=${CUEBOT_GRPC_RQD_SERVER_PORT:50051}
 grpc.rqd_cache_size=500
 # RQD Channel Cache Expiration in Minutes
 grpc.rqd_cache_expiration=30
+
+dispatcher.job_query_max=20
+dispatcher.job_lock_expire_seconds=5
```

Comment:
Should we lower this default value?

Reply:
Lowered to 2 for testing. 0 for default deployment.